### PR TITLE
Fix ckeditor mirrored img srcsets

### PR DESCRIPTION
**Pull request by s-cheng**
This was reported by Lorenzo via [slack](https://cea-core.slack.com/archives/C027Z3794F3/p1696758716846219):

> Images sometimes don't show up in comments after they're posted, but they come back if I click "edit" and then stay after I click "save"

Lorenzo also provided the img tag in slack. Seems like the issue is that, when we are mirroring images, we don't account for commas in the srcset URLs and accidentally insert extra spaces. I didn't actually find the right path to reproduce this issue but I think this should fix it.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1205848127005747) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by s-cheng** [comment]
@jimrandomh FYI (since we worked on this together a while back)

**Comment by jpaddison3** [comment]
I'm confused, this is in a 1 off script, right?

**Comment by s-cheng** [comment]
@jpaddison3 Oh no sorry this doesn't affect existing broken srcsets, I'm just fixing what I think is the actual bug. Lorenzo said that just editing & saving the comment fixed the images.

File: packages/lesswrong/server/scripts/convertImagesToCloudinary.ts [hunk 1]
```diff
@@ -159,7 +159,7 @@ function getImageUrlsFromImgTag(tag: any): string[] {
   }
   const srcset: string = tag.attr("srcset");
   if (srcset) {
-    const imageVariants = srcset.split(",").map(tok=>tok.trim());
+    const imageVariants = srcset.split(", ").map(tok=>tok.trim());
```
Comment (jpaddison3) [inline packages/lesswrong/server/scripts/convertImagesToCloudinary.ts:162]:
```suggestion
    const imageVariants = srcset.split(",\s").map(tok=>tok.trim());
```

Would it be valid html have only a newline after the comma?

Comment (jpaddison3) [reply packages/lesswrong/server/scripts/convertImagesToCloudinary.ts:162]: And below

**Review by jpaddison3 — APPROVED** [review]
As I was saying before github so rudely gave me a 500 error on Friday.

**Pull request merged**